recipe/meta.yaml

```diff
@@ -9,6 +9,9 @@ source:
   url: https://cmake.org/files/v3.5/cmake-3.5.0-win32-x86.zip  # [win]
   fn: cmake-3.5.0-win32-x86.zip # [win]
 
+build:
+  number: 1
```

Comment:
I'm really not in favour of bumping this each time we add a maintainer - the downstream impact is greater than the value it adds IMO (more built distributions on anaconda.org, more user downloads, more packages in users package cache, lower fetch and resolve speeds (marginal) etc.).

I'm not saying carte blanche that we *shouldn't* do it, just that it needn't happen each and every time somebody would like to contribute to the maintenance. Heck, it may be that by adding a maintainer, we get some updates anyway (version bumps, compiler options etc.) - maybe we should encourage it in that way :wink: